frontend/src/modules/controllers/ValueClicker.tsx

```diff
@@ -98,7 +98,13 @@ export const ValueClicker = ({
     const change = value + step;
     setDisableIncrement(change >= max);
     setDisableDecrement(change <= min);
-    return onClick(change);
+    if (change >= max) {
+      return onClick(max);
+    } else if (change <= min) {
+      return onClick(min);
+    } else {
+      return onClick(change);
+    }
```

Comment:
```suggestion
    if (change >= max) {
      return onClick(max);
    }
    if (change <= min) {
      return onClick(min);
    }
    return onClick(change);
```

Reply:
I guess it warns even if I have else if's with the same return statements.